src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""The following are the unpolarized time-like leading order Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """
+    Computes the LO quark-quark anomalous dimension
+    Implements Eqn. (B.3) from hep-ph/0604160
+
+    Input parameters
+    ----------------
+    N : Mellin moment (type: complex)
```

Comment:
Start the description on a new line, the line continuation is reserved for the (optional) type.

In this case put the type: in principle we are trying to avoid, and use [type hints](https://docs.python.org/3/library/typing.html), but in this case is `numba` decorated (the `nb.njit`), better to avoid, and just describe in the docstring.
```suggestion
    N : complex
        Mellin moment (type: complex)
```